testProjects/plugins/removePrint.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
```

Comment:
This file should be excluded from the project since it's transpiled from `removePrint.ts`